google/src/main/scala/org/broadinstitute/dsde/rawls/google/GoogleUtilities.scala

```diff
@@ -36,11 +36,15 @@ trait GoogleUtilities extends LazyLogging with Retry {
     }
   }
 
-  protected def retryWhen500orGoogleError[T](op: () => T) = {
+  protected def retryWhen[T](pred: Predicate[Throwable])(op: () => T): Future[T] = {
```

Comment:
This doesn't seem to be referenced anywhere.  Did you intend to compose the other `retryWhen`s using this?